tasks/common/requestHelper.ts

```diff
@@ -275,17 +276,18 @@ export function withRetry<T>(
     });
 }
 
-/** Indicates whether the given object is an HTTP response for a 4xx error. */
-export function is400Error(err): boolean
+/** Indicates whether the given object is an HTTP response for a retryable error. */
+export function isRetryableError(err): boolean
 {
     // Does this look like a ResponseInformation?
     if (err != undefined && err.response != undefined && typeof err.response.statusCode == 'number')
     {
-        return err.response.statusCode >= 400
-            && err.response.statusCode < 500
+        return err.response.statusCode == 429 // 429 code is returned by the API for throttle down. This is retriable
```

Comment:
Minor: For comment, instead of explaining 429, consider this: 
See here for 429 HTTP Status Code : https://tools.ietf.org/html/rfc6585#section-4